packages/iov-ethereum/src/responses.ts

```diff
@@ -0,0 +1,11 @@
+import { ReadonlyDate } from "readonly-date";
+
+import { ChainId } from "@iov/base-types";
+
+export interface Header {
```

Comment:
Can we call this `BlockHeader`? I know "header" was the name used so far but I think this is not descriptive enough for a newcomers to find their way. In https://github.com/iov-one/iov-core/issues/570 I'll use "block header" for all interfaces.